### PR TITLE
Cleanup lisk_bridge.sh

### LINKED ISSUE #64: Clean up lisk_bridge.sh script
* Make lisk_bridge.sh a downloadable script
* Remove BRIDGE_HOME
* Use jq to parse values out of json files

### PR DESCRIPTION
**Pull request by Nazgolze**
Closes #64

### CODE REVIEW

File: downloaded/lisk_bridge.sh [hunk 1]
```diff
@@ -24,30 +24,32 @@
 TARGET_HEIGHT="3513100"
```
[the remark this answers is not in the document]
Comment (Nazgolze) [reply downloaded/lisk_bridge.sh]: True. We probably shouldn't have a default and should make specifying it as an argument required

File: downloaded/lisk_bridge.sh [hunk 2]
```diff
@@ -21,45 +21,46 @@
...
-			 b) BRIDGE_HOME="$OPTARG" ;; # Where the bridge is located
-			 n) BRIDGE_NETWORK="$OPTARG" ;; # Which network is being bridged
-			 h) TARGET_HEIGHT="$OPTARG" ;; # What height to cut over at
-		 esac
-	 done
+			s) LISK_HOME="$OPTARG" ;;
+			n) BRIDGE_NETWORK="$OPTARG" ;; # Which network is being bridged
+			h) TARGET_HEIGHT="$OPTARG" ;; # What height to cut over at
+			:) echo 'Missing argument for -'"$OPTARG" >&2;
+			   SHOW_USAGE=1;;
```
Comment (fchavant) [inline downloaded/lisk_bridge.sh]: Are we setting this variable to not duplicate code?
Not sure this is worth it (1 line here + 4 lines below vs. 2 lines)

File: downloaded/lisk_bridge.sh [hunk 3]
```diff
@@ -21,45 +21,46 @@
...
 }
 
 # Harvests the configuation data from the source installation
 # for an automated cutover.
 extractConfig() {
-	PM2_CONFIG="$LISK_HOME/etc/pm2-lisk.json"
-	LISK_CONFIG="$(grep "config" "$PM2_CONFIG" | cut -d'"' -f4 | cut -d' ' -f2)" >> /dev/null
-	LISK_CONFIG="$LISK_HOME/$LISK_CONFIG"
+	LISK_CONFIG="$LISK_HOME/config.json"
 	export PORT
```
Comment (fchavant) [inline downloaded/lisk_bridge.sh]: Why create a function (that gets called only once) if all is does is `export` one variable?

File: downloaded/lisk_bridge.sh [hunk 4]
```diff
@@ -88,7 +97,7 @@ while [[ "$BLOCK_HEIGHT" -lt "$TARGET_HEIGHT" ]] ; do
 done
 
 cd "$LISK_HOME" || exit 2
```
Comment (fchavant) [inline downloaded/lisk_bridge.sh]: Not necessary.

File: downloaded/lisk_bridge.sh [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+	echo -e '\nExample: bash lisk_bridge.sh -h 50000000 -n test -s /home/lisk/lisk-test'
+	exit 1;
+fi
+if [[ ! $LISK_HOME ]]; then
+	LISK_HOME="$HOME/lisk-$BRIDGE_NETWORK"
+fi
+JQ="$LISK_HOME/bin/jq"
+
+LISK_CONFIG="$LISK_HOME/config.json"
+export PORT
```
Comment (fchavant) [inline downloaded/lisk_bridge.sh]: I don't think this `export` is needed.

**Pull request merged**